add doc for strict login checking required for disabling login with LDAP email address (#2819)
owncloud/docs: modules/admin_manual/pages/configuration/user/user_auth_ldap.adoc

LDAP Email Address::
  If this value is checked, the login value will be compared to an email address in the LDAP directory; specifically, the `mailPrimaryAddress` and `mail` attributes.

IMPORTANT: Disallowing login with LDAP Email Address requires enabling strict login checking to be effective:

>   

Reviewer: ```suggestion```

>   

Reviewer: ```suggestion```

LDAP Email Address::
  If this value is checked, the login value will be compared to an email address in the LDAP directory; specifically, the `mailPrimaryAddress` and `mail` attributes.

>   IMPORTANT: Disallowing login with LDAP Email Address requires enabling strict login checking to be effective: **sudo -u www-data php occ config:system:set --type boolean --value true strict_login_enforced**

Reviewer: ```suggestion
IMPORTANT: Disallowing login with LDAP Email Address requires enabling strict login checking to be effective:
  
[source,console,subs="attributes+"]
----
{occ-command-example-prefix} config:system:set --type boolean --value true strict_login_enforced
----
```

LDAP Email Address::
  If this value is checked, the login value will be compared to an email address in the LDAP directory; specifically, the `mailPrimaryAddress` and `mail` attributes.

> IMPORTANT: Disallowing login with LDAP Email Address requires enabling strict login checking to be effective:

Reviewer: ```suggestion
IMPORTANT: Disallowing login with LDAP Email Address requires enabling strict login checking to be effective: +
```

>   

Reviewer: ```suggestion
```
